frontend/src/components/print/print-react/DownloadReactPdfListItem.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <v-list-item @click="generatePdf">
+    <v-list-item-icon>
+      <v-icon v-if="loading" class="mdi-spin">mdi-loading</v-icon>
+      <v-icon v-else>mdi-react</v-icon>
+    </v-list-item-icon>
+    <v-list-item-title>
+      {{ $tc("components.print.localPdfDownloadButton.label") }}
+    </v-list-item-title>
+  </v-list-item>
+</template>
+
+<script>
+import { saveAs } from 'file-saver'
+import slugify from 'slugify'
+
+const RENDER_IN_WORKER = true
+
+export default {
+  name: 'DownloadReactPdfListItem',
+  props: {
+    config: {
+      type: Object,
+      default: () => {}
+    }
+  },
+  data () {
+    return {
+      loading: false
+    }
+  },
+  methods: {
+    async generatePdf () {
+      if (this.loading) {
+        return
+      }
+
+      this.loading = true
+
+      // lazy load generatePdf to avoid loading complete react-pdf when showing PDF download buton
```

Comment:
```suggestion
      // lazy load generatePdf to avoid loading complete react-pdf when showing PDF download button
```
(typo also exists in the button component)

Reply:
Fixed in fcee4dc9